### PR TITLE
Updating the contributing.md 

### PR DESCRIPTION
**Pull request by LHinson**
I have updated the contributing.md to include information about how to become a maintainer.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,6 +16,10 @@
   - [React Component Requirements](#react-component-requirements)
   - [Code Consistency](#code-consistency)
   - [Code Contribution Guidelines](#code-contribution-guidelines)
+- [Becoming a Maintainer](#becoming-a-maintainer)
+  - [How do I become a maintainer?](how-do-i-become-a-maintainer?)
```
Comment (ohadlevy) [inline CONTRIBUTING.md]: maintainer or committer? there might be a slight difference between the two

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -233,6 +237,58 @@ Once your pull request has been reviewed, if all conditions above have been met
...
+
+- Review and merge code and documentation.
+- Help triaging bugs and testing pull requests.
+- Make well formed pull requests.
+- Have a sense of duty about the project.
+- Play well with others, are respectful, show gratitude.
+
+If you want to become a maintainer, we expect you to:
+
+- Review and test pull requests submitted by others .
```
Comment (ohadlevy) [inline CONTRIBUTING.md]: nitpick: extra space EOL before the .

File: CONTRIBUTING.md (hunk 1 above)
Comment (LHinson) [reply CONTRIBUTING.md]: @ohadlevy yeah I noticed this discrepancy between Foreman and PF. For PatternFly, each repo has two sets of teams, "Contributors" and "Maintainers". Therefore I decided to keep consistent with our current definitions as seen on our teams page: https://github.com/orgs/patternfly/teams

WDYT?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -233,6 +237,58 @@ Once your pull request has been reviewed, if all conditions above have been met
...
+
+### How do I lose maintainers status?
+
+If you are inactive in the community for one year, we will remove you from the maintainers list and revoke your permission, but we will make a mention of you on a  list of previous maintainers.
+
+In the event that a maintainer continues to disregard good citizenship (or actively disrupts the project), we may need to revoke that person’s status. The process is the same as for nominating a new maintainer: someone suggests the revocation with a good reason, two people second the motion, and a vote may be called if consensus cannot be reached. We hope that’s simple enough, and that we never have to test it in practice.
+
+### Quick tips for new maintainers
+
+- If something you merged broke something, it’s your responsibility to provide a fix.
```
Comment (dlabrecq) [inline CONTRIBUTING.md]: I don't agree with this how this statement is worded. I believe this applies more to a committer, not necessarily the person merging a PR.

It's typically not the maintainer's responsibility to provide fixes for code they did not write? The person who wrote the code really should be the one to provide a fix.

Perhaps it should be the maintainer's responsibility to back out a commit that caused the breakage? That is, until the developer of the code can provide a fix.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -233,6 +237,58 @@ Once your pull request has been reviewed, if all conditions above have been met
...
+
+Such objections may happen in public on the nomination e-mail thread. However, not everyone is comfortable giving objections publicly. Therefore, it is acceptable for other maintainers to raise their concerns with the sponsor and/or other maintainers privately if they wish to do so. The sponsor is expected to update the nomination e-mail thread to show that it is on hold pending private concerns.
+
+Regardless, while objections are discussed either in private or public, the nomination is on hold until the objections are resolved or the nomination is rejected. In the event of a failed nomination, the sponsor (as part of the discussing group) will know the grounds for the rejection, and can pass along constructive feedback to the candidate. Care should be taken to do this sensitively.
+
+### How do I lose maintainers status?
+
+If you are inactive in the community for one year, we will remove you from the maintainers list and revoke your permission, but we will make a mention of you on a  list of previous maintainers.
```
Comment (dlabrecq) [inline CONTRIBUTING.md]: One year is very generous -- I'm thinking 6 months.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -233,6 +237,58 @@ Once your pull request has been reviewed, if all conditions above have been met
...
+
+In the event that a maintainer continues to disregard good citizenship (or actively disrupts the project), we may need to revoke that person’s status. The process is the same as for nominating a new maintainer: someone suggests the revocation with a good reason, two people second the motion, and a vote may be called if consensus cannot be reached. We hope that’s simple enough, and that we never have to test it in practice.
+
```
Comment (dlabrecq) [inline CONTRIBUTING.md]: The process for nominating a person sounds good, but the same process for revoking? I can see that revoking a persons privileges publicly could get ugly. This should probably be done more discreetly so people are not publicly shamed.

**Comment by dtaylor113** [comment]
Not part of the changes for this PR, just general comment for Contribution Process -> 
Creating Issues for Bugs:
> 1. In the issue tracker, check if the bug has already been reported.
If it does exist, ...add a comment to the existing bug.

For existing bugs, OpenShift/Bugzilla also have a practice of updating a counter, or adding `+1`', whenever a dev/user hits the bug.  Good indication of frequency of occurrence.

File: CONTRIBUTING.md (hunk 3 above)
Comment (dtaylor113) [reply CONTRIBUTING.md]: Perhaps better worded as "All maintainers are responsible for the health of the repo.  If unit tests, doc/example site, or CI (travis, jenkins) fails; you are responsible for investigating and/or providing a fix".

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -233,6 +237,58 @@ Once your pull request has been reviewed, if all conditions above have been met
...
+- Make well formed pull requests.
+- Have a sense of duty about the project.
+- Play well with others, are respectful, show gratitude.
+
+If you want to become a maintainer, we expect you to:
+
+- Review and test pull requests submitted by others.
+- Encourage and ensure design remains an integral part of the review process and pull in designers for review as needed (you can leverage @patternfly/pf-<framework>-design if there is no known associated designer).
+- Maintain sustained activity versus sporadic.
+- Support users and other developers on [PatternFly Slack](https://patternfly.slack.com/) and the [mailing list](mailto:patternfly@redhat.com).
```
Comment (dtaylor113) [inline CONTRIBUTING.md]: Probably want to add the pf-react slack channel and mailing list.

Comment (LHinson) [reply CONTRIBUTING.md]: There is no react ML. I will update the slack info though.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -233,6 +237,58 @@ Once your pull request has been reviewed, if all conditions above have been met
...
+- Review and merge code and documentation.
+- Help triaging bugs and testing pull requests.
+- Make well formed pull requests.
+- Have a sense of duty about the project.
+- Play well with others, are respectful, show gratitude.
+
+If you want to become a maintainer, we expect you to:
+
+- Review and test pull requests submitted by others.
+- Encourage and ensure design remains an integral part of the review process and pull in designers for review as needed (you can leverage @patternfly/pf-<framework>-design if there is no known associated designer).
```
Comment (jgiardino) [inline CONTRIBUTING.md]: `@patternfly/pf-<framework>-design` should be updated to `@patternfly/patternfly-react-ux`

File: CONTRIBUTING.md (hunk 3 above)
Comment (jgiardino) [reply CONTRIBUTING.md]: I don't object to @dtaylor113's comment, but I also like being explicit about what to do if you merged a PR and it broke something. I agree with Dan's comment that the person merging might not be the best person to fix the issue, and we should give them the option to back out a commit.

File: CONTRIBUTING.md (hunk 5 above)
Comment (jgiardino) [reply CONTRIBUTING.md]: Similar to Dan's comment, if I can privately object to a nomination, then should I be able to privately request that their status be removed? If that's the case, who would I send my request to?

File: CONTRIBUTING.md (hunk 3 above)
Comment (priley86) [reply CONTRIBUTING.md]: yes - this has happened in the past and during high activity periods, i think a judgement call should be made about other contributors submitting a fix (if the committer can't provide one immediately). But usually it is just an open discussion...but yes, agree w/ @dtaylor113 ' s point about all maintainers trying to retain the health of the repo at all times. The test coverage/CI tooling in this repo is steadily improving so hopefully this is less and less of an issue..

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -233,6 +237,58 @@ Once your pull request has been reviewed, if all conditions above have been met
...
+### How do I lose maintainers status?
+
+If you are inactive in the community for one year, we will remove you from the maintainers list and revoke your permission, but we will make a mention of you on a  list of previous maintainers.
+
+In the event that a maintainer continues to disregard good citizenship (or actively disrupts the project), we may need to revoke that person’s status. The process is the same as for nominating a new maintainer: someone suggests the revocation with a good reason, two people second the motion, and a vote may be called if consensus cannot be reached. We hope that’s simple enough, and that we never have to test it in practice.
+
+### Quick tips for new maintainers
+
+- If something you merged broke something, it’s your responsibility to provide a fix.
+- Do not merge your own commits
```
Comment (priley86) [inline CONTRIBUTING.md]: does this tie in to the existing approval process agreed upon in pf-react? I guess it is good to distinguish that we don't self-merge when one becomes a maintainer...and we typically wait for ux/design/development review (unless a PR doesn't pertain such as tests/build/tooling changes/dependency changes only). This repo is mission critical now for many teams so it's important to keep it healthy as we introduce new components/new build processes/new tooling/new workspaces for development. I believe that additional maintainers will help further this and continue to improve the quality overall, as well as the performance downstream.

**Comment by priley86** [comment]
exciting - thanks @LHinson 🌟

**Comment by LHinson** [comment]
Thanks everyone for the awesome review! @jgiardino @dtaylor113 @dlabrecq @priley86 @ohadlevy @jeff-phillips-18 I've made updates per your recommendations. 

@dtaylor113 I didn't update the part about code since I'm not so knowledgable about this. File an issue to update this in a separate PR.

**Pull request merged**